assets/app/scripts/directives/logViewer.js

```diff
@@ -93,19 +93,21 @@ angular.module('openshiftConsole')
               } else {
                 // If the user scrolled the window manually, stop auto-scrolling.
                 $scope.$evalAsync(function() {
-                  $scope.autoScroll = false;
+                  $scope.autoScrollIsActive = false;
```

Comment:
Yeah, these are getting long... but the little bugs have pushed me to be more and more specific :)
Ill update that.